public void testInitNullCallable()
public void testInitExecutor()
public void testInitExecutorNullCallable()
public void testInitialize() throws Exception
public Integer call() throws Exception
Integer RESULT=Optional[Integer.valueOf(42)]
